src/bin/tikv-server.rs

```diff
@@ -584,7 +594,17 @@ fn run_raft_server(listener: TcpListener,
                           resolver,
                           snap_mgr)
         .unwrap();
-    start_server(svr, event_loop, engine);
+
+    let backup_path = {
+        let mut path = get_toml_string(config, "server.backup", Some(String::new()));
+        if path.is_empty() {
+            path = store_path;
```

Comment:
If `store_path="/somepath/data/"`, then `backup_path="/somepath/data/_backup/"`, if `store_path="/somepath/data"`, then `backup_path="/somepath/data_backup/"`, is it except?